single_stage_detector/README.md

```diff
@@ -0,0 +1,293 @@
+`config_DGXA100_032x08x032.sh` scripts.
+
+Steps required to launch multi node training on NVIDIA DGX-A100:
+
+```
+cd ./single_stage_detector/ssd
+source config_DGXA100_008x08x008.sh # or config_DGXA100_032x08x032.sh
+DATADIR=<path/to/data/dir> LOGDIR=<path/to/output/dir> sbatch -N $DGXNNODES -t $WALLTIME run.sub
+```
+
+## Hyperparameter settings
+Hyperparameters are recorded in the `config_*.sh` files for each configuration
+and in `run_and_time.sh`.
+
+# Dataset/Environment
+## Publication/Attribution
+[Google Open Images Dataset V6](https://storage.googleapis.com/openimages/web/index.html)
+
+## The MLPerf Subset
+The MLPerf subset includes only 264 classes of the total 601 available in the
+full dataset:
+
+| Dataset           | # classes | # train images | # validation images | Size  |
+|-------------------|-----------|----------------|---------------------|-------|
+| OpenImages Full   | 601       | 1,743,042      | 41,620              | 534GB |
+| OpenImages MLperf | 264       | 1,170,301      | 24,781              | 352GB |
+
+The list of used classes can be viewed
+[here](https://github.com/mlcommons/training/blob/master/single_stage_detector/scripts/download_openimages_mlperf.sh).
+
+# Model
+This network takes an input 800x800 image from [OpenImages-v6](https://storage.googleapis.com/openimages/web/index.html)
+and 265 categories, and computes a set of bounding boxes and categories.
+Other detector models use multiple stages, first proposing regions of interest
+that might contain objects, then iterating over the regions of interest to try
+to categorize each object. SSD does both of these in one stage, leading to
+lower-latency and higher-performance inference.
+
+## Backbone
+
+The backbone is based on ResNeXt50_32x4d as described in Section 3 of
+[this paper](https://arxiv.org/pdf/1611.05431.pdf).  Using the
+same notation as Table 1 of the paper the backbone looks like:
+
+| stage      | # stacked blocks | shape of a residual block  |
+| :--------: | :--------------: | :------------------------: |
+| conv1      |                  | 7x7, 64, stride 2          |
+|            |                  | 3x3 max pool, stride 2     |
+| conv2_x    | 3                | 1x1, 128                   |
+|            |                  | 3x3, 128, groups=32        |
+|            |                  | 1x1, 256                   |
+| conv3_x    | 4                | 1x1, 256                   |
+|            |                  | 3x3, 256, groups=32        |
+|            |                  | 1x1, 512                   |
+| conv4_x    | 6                | 1x1, 512                   |
+|            |                  | 3x3, 512, groups=32        |
+|            |                  | 1x1, 1024                  |
+| conv5_x    | 3                | 1x1, 1024                  |
+|            |                  | 3x3, 1024, groups=32       |
+|            |                  | 1x1, 2048                  |
+
```

Comment:
I think it will be useful to indicate somewhere here that only the last 3 stages of the backbone are trainable... also would be useful to link to the part of the code where this logic (whether a stage is trainable or not) is implemented...
for e.g..
https://github.com/ahmadki/training/blob/akiswani/ssd_v2/single_stage_detector/ssd/model/backbone_utils.py#L96

Reply:
done